course7/homeworks/nadinasovaiala/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
This is too high. Someone with CMake 3.1 could easily use it, because the file doasn't xcontain any super-new Cmake features, but you prevent this.
Use 3.0 at the most